ooc-cli.use

```diff
@@ -0,0 +1,6 @@
+Name: ooc CLI
+Description: CLI helpers
+SourcePath: source/cli
+Requires: ooc-collections
```

Comment:
The only thing it does for us is indicating what other module this module depends on. I think it is used by a tool called [sam](https://ooc-lang.org/docs/tools/sam/).